Mathlib/NumberTheory/Transcendental/Lindemann/Init/AnalyticalPart.lean

```diff
@@ -0,0 +1,206 @@
+/-
+Copyright (c) 2022 Yuyang Zhao. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Yuyang Zhao
+-/
+import Mathlib.Algebra.Polynomial.SumIteratedDerivative
+import Mathlib.Analysis.Calculus.Deriv.Polynomial
+import Mathlib.Analysis.SpecialFunctions.ExpDeriv
+import Mathlib.MeasureTheory.Integral.FundThmCalculus
+import Mathlib.RingTheory.Int.Basic
+import Mathlib.Topology.Algebra.Polynomial
+
+/-!
+# The Lindemann-Weierstrass theorem
+-/
```

Comment:
This needs a bit more words... As stated, it looks like this file proves the whole theorem.